internal/tuf/v01/root.go

```diff
@@ -784,3 +786,108 @@ func (o *OtherRepository) GetInitialRootPrincipals() []tuf.Principal {
 	}
 	return initialRootPrincipals
 }
+
+// Hooks defines the schema that groups hooks by stage.
+type Hooks struct {
+	PreCommit map[string]Hook `json:"pre-commit"`
```

Comment:
```suggestion
	PreCommit map[string]*Hook `json:"pre-commit"`
```

Reply:
Also don't love the use of `-` in the json tag, maybe we can camelcase it `preCommit` for consistency?